crypto/s2n_hash.c

```diff
@@ -568,6 +604,10 @@ int s2n_hash_reset(struct s2n_hash_state *state)
 
 int s2n_hash_free(struct s2n_hash_state *state)
 {
+    if (state == NULL)
+    {
+        return S2N_SUCCESS;
+    }
```

Comment:
Is that causing the problem on CI? @camshaft

Reply:
Hmmm... possibly? Easy way to check is to remove it and see if it passes :)